spec/locales-currencies-tz.html

```diff
@@ -164,59 +164,229 @@ <h1>
+        For example, *"Atlantic/Jan_Mayen"* must resolve to *"Arctic/Longyearbyen"*.
+      </li>
+      <li>
+        <emu-not-ref>Legacy</emu-not-ref> POSIX Zone names must be non-primary time zone identifiers that resolve to their closest non-legacy equivalents, as shown in the table below:
+      </li>
+    </ul>
+
+    <emu-table id="table-posix-time-zone-identifier-mapping">
+      <emu-caption><emu-not-ref>Legacy</emu-not-ref> POSIX Time Zone Identifier Mapping</emu-caption>
+      <table class="real-table">
+        <thead style="white-space: nowrap">
+          <tr>
+            <th><emu-not-ref>Legacy</emu-not-ref> POSIX Zone Name</th>
+            <th>Primary Time Zone Identifier</th>
+          </tr>
+        </thead>
+        <tr>
+          <td>*"EST"*</td>
+          <td>*"Etc/GMT+5"*</td>
+        </tr>
+        <tr>
+          <td>*"MST"*</td>
+          <td>*"Etc/GMT+7"*</td>
+        </tr>
+        <tr>
+          <td>*"HST"*</td>
+          <td>*"Etc/GMT+10"*</td>
+        </tr>
+        <tr>
+          <td>*"EST5EDT"*</td>
+          <td>*"America/New_York"*</td>
+        </tr>
+        <tr>
+          <td>*"CST6CDT"*</td>
+          <td>*"America/Chicago"*</td>
+        </tr>
+        <tr>
+          <td>*"MST7MDT"*</td>
+          <td>*"America/Denver"*</td>
+        </tr>
+        <tr>
+          <td>*"PST8PDT"*</td>
+          <td>*"America/Los_Angeles"*</td>
+        </tr>
+        <tr>
+          <td>*"WET"*</td>
+          <td>*"Europe/Lisbon"*</td>
+        </tr>
+        <tr>
+          <td>*"CET"*</td>
+          <td>*"Europe/Berlin"*</td>
+        </tr>
+        <tr>
+          <td>*"MET"*</td>
+          <td>*"Europe/Vienna"*</td>
+        </tr>
+        <tr>
+          <td>*"EET"*</td>
+          <td>*"Europe/Athens"*</td>
+        </tr>
+      </table>
+    </emu-table>
+
+    <emu-note>
+      <p>
+        The IANA Time Zone Database offers build options that affect which time zone identifiers are primary.
+        The default build options merge different countries' time zones, for example *"Atlantic/Reykjavik"* is built as a Link to the Zone *"Africa/Abidjan"*.
+        Geographically and politically distinct locations are likely to introduce divergent time zone rules in a future version of the IANA Time Zone Database.
+        The exceptions above serve to mitigate these future-compatibility issues for ECMAScript programmers and end users.
+      </p>
+      <p>
+        International Components for Unicode (ICU) <a href="https://icu.unicode.org/">https://icu.unicode.org/</a> is a widely-used library that exposes IANA Time Zone Database information.
```

Comment:
Refer to cldr first here. That xml file is not "ICU's". 

Then afterwards say for example ICU is widely used etc.

Reply:
Per other comments I ended up just focusing on CLDR here, and removing the reference to ICU because the data in question is wholly contained in CLDR.